src/wp-includes/class-wp-user-query.php

```diff
@@ -1043,10 +1045,11 @@ protected function parse_orderby( $orderby ) {
 	 * Generate cache key.
 	 *
 	 * @since 6.3.0
+	 * @since 6.9.0 $args is no longer used.
 	 *
 	 * @global wpdb $wpdb WordPress database abstraction object.
 	 *
-	 * @param array  $args Query arguments.
+	 * @param array  $args Unused. Query arguments.
```

Comment:
```suggestion
	 * @param array  $deprecated Unused.
```

I can't apply suggestions throughout the function as some parts are outside the diff.